src/webgpu/idl/render_pass/storeOp.spec.ts

```diff
@@ -0,0 +1,14 @@
+export const description = `IDL Tests for RenderPass StoreOp.
+
+  Test Coverage Needed:
+
+  - Test that a color attachment's storeOp is store by default.
+
+  - Test that depth/stencil attachment's depthStoreOp is store by default.
+
+  - Test that depth/stencil attachment's stencilStoreOp is store by default.`;
+
+import { makeTestGroup } from '../../../common/framework/test_group.js';
+import { IDLTest } from '../idl_test.js';
+
+export const g = makeTestGroup(IDLTest);

```

Comment:
FYI there's an issue right now where IDLTests are not GPUTests so you can't actually test creating anything. I was meaning to fix this, but anyone can fix it anytime.

That said, the IDL tests are less important (they're a lot less nuanced than the validation tests, and already have a standardized language backing them so the implementations should be mostly consistent by default), so okay to put these tests off. I was also considering whether we could partially automate these to some extent, so probably at some point someone should sit down and focus on just the IDL tests and figure out the right way to do them.

Reply:
IDL tests were moved into /operation/, so not going to worry about this.